python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -259,7 +307,7 @@ def getStateTime(id, status):
         checkstr = ''
 
     if checkstr == '':
-        sys.stdout.write(None)
+        print("%s" % None)
```

Comment:
I prefer this, that probably works as well, although I don't think we test these functions properly